publication/src/main/scala/hmda/publication/reports/disclosure/D51.scala

```diff
@@ -0,0 +1,155 @@
+package hmda.publication.reports.disclosure
+
+import java.util.Calendar
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import hmda.model.publication.reports.ApplicantIncomeEnum._
+import hmda.publication.reports._
+import hmda.model.publication.reports.ReportTypeEnum.Disclosure
+import hmda.model.publication.reports._
+import hmda.publication.reports.util.DateUtil._
+import hmda.publication.reports.util.DispositionType._
+import hmda.publication.reports.util.ReportUtil._
+import hmda.query.model.filing.LoanApplicationRegisterQuery
+
+import scala.concurrent.Future
+
+case class D51(
+  respondentId: String,
+  institutionName: String,
+  table: String,
+  reportType: ReportTypeEnum,
+  description: String,
+  year: Int,
+  reportDate: String,
+  msa: MSAReport,
+  applicantIncomes: List[ApplicantIncome],
+  total: List[Disposition]
+)
+
+object D51 {
+  def apply(
+    respondentId: String,
+    institutionName: String,
+    year: Int,
+    reportDate: String,
+    msa: MSAReport,
+    applicantIncomes: List[ApplicantIncome],
+    total: List[Disposition]
+  ): D51 = {
+
+    val description = "Disposition of applications for FHA, FSA/RHS, and VA home-purchase loans, 1- to 4-family and manufactured home dwellings, by income, race and ethnicity of applicant"
```

Comment:
Given that we're going to have dozens of these edits, would it be a good idea to move these kinds of Strings to a separate file, similar to the `EditMetaData`?